bot/exts/moderation/silence.py

```diff
@@ -331,6 +340,12 @@ async def _unsilence(self, channel: TextOrVoiceChannel) -> bool:
             role = self._everyone_role
             overwrite = channel.overwrites_for(role)
             permissions = "`Send Messages` and `Add Reactions`"
+        elif isinstance(channel, Thread):
+            await channel.edit(locked=False)
```

Comment:
Same as above: if the thread was manually unlocked before the scheduled time, there won't be any indication of it.